src/lib/style/components/_project-view-card.scss

```diff
@@ -10,7 +10,15 @@
   text-align: left;
 
   img {
+    display: block;
     max-width: 100%;
+    max-height: 30%;
+    width: auto;
+    height: auto;
```

Comment:
Unless somewhere else in the codebase is setting the width and height for the <img> tag, their default values should just be `auto`. See the Formal Definition section on [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width) & [height](https://developer.mozilla.org/en-US/docs/Web/CSS/height). But it's okay to leave it too in case you want to enforce the CSS rules.

Reply:
You're right. I will remove these.